text/0000-no-more-extern-crate.md

```diff
@@ -70,29 +85,21 @@ For example, `cargo build`-ing a crate `my_crate` with a dependency on `rand`
 results in a call to rustc that looks something like
 `rustc --crate-name mycrate src/main.rs --extern rand=/path/to/librand.rlib ...`.
 
-When an external crate is specified this way,
-the crate will automatically brought into scope as if an
-`extern crate name_of_crate;`
-declaration had been added to the current crate root.
-This behavior won't occur when including a library using the `-l`
+When an external crate is specified this way, it will be automatically
+available to any module in the current crate through `use` statements or
+absolute paths (e.g. `::rand::random()`). It will _not_ be automatically
+imported at root level as happens with current `extern crate`.
```

Comment:
>if absolute paths work, relative paths at the root level should work for the system to be self-consistent

There's a precedent however, when this doesn't happen, not at the root level though.
```rust
enum E {
    U,
    V = E::U, // OK, resolved
    W = U, // ERROR, cannot find value `U` in this scope
}
```
This should be equally confusing, it just comes up very rarely.